ignite/contrib/handlers/lr_finder.py

```diff
@@ -187,7 +187,7 @@ def get_results(self) -> Dict[str, List[Any]]:
         return self._history
 
     def plot(
-        self, skip_start: int = 10, skip_end: int = 5, log_lr: bool = True, filepath: str = None, **kwargs: Any,
+        self, skip_start: int = 10, skip_end: int = 5, log_lr: bool = True, filepath: str = "", **kwargs: Any,
```

Comment:
```suggestion
        self, skip_start: int = 10, skip_end: int = 5, log_lr: bool = True, filepath: Optional[str] = None, **kwargs: Any,
```
Let's use optional instead.